### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

File: docs/src/main/asciidoc/README.adoc [hunk 1]
```diff
@@ -0,0 +1,238 @@
+:repo_url: https://github.com/marcingrzejszczak/database-update
+
+= Zero Downtime Deployment Database Issues
+
+{repo_url}
+
+This article will explain in more depth how to tackle issues related to database compatibility with Spring Boot and
```
Comment (dsyer) [inline docs/src/main/asciidoc/README.adoc]: You don't need to mention Spring Boot this early. It's a fairly general topic, using a specific example with Spring Boot (which you could explain at the end of this paragraph, for instance).

File: docs/src/main/asciidoc/README.adoc [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+database versioning tooling. We will present what can happen with your production applications if you try to perform
+such a deployment unprepared. We will also walk through the lifecycle of an application in order to perform zero
+downtime deployment and at the end of day apply a backwards incompatible database change in a backwards compatible way.
```
Comment (dsyer) [inline docs/src/main/asciidoc/README.adoc]: "at the end of the day" seems like the wrong idiom here. You probably should start a new sentence and write "And finally, we will see how to ..."? Or something.

File: docs/src/main/asciidoc/README.adoc [hunk 3]
```diff
@@ -0,0 +1,238 @@
+:repo_url: https://github.com/marcingrzejszczak/database-update
+
+= Zero Downtime Deployment Database Issues
+
+{repo_url}
+
+This article will explain in more depth how to tackle issues related to database compatibility with Spring Boot and
+database versioning tooling. We will present what can happen with your production applications if you try to perform
+such a deployment unprepared. We will also walk through the lifecycle of an application in order to perform zero
```
Comment (dsyer) [inline docs/src/main/asciidoc/README.adoc]: also->then?

Suggest something like this: "We will then wall through the steps in the lifecycle of an application that are necessary in order to have zero downtime."

File: docs/src/main/asciidoc/README.adoc (hunk 1 above)
Comment (dsyer) [inline docs/src/main/asciidoc/README.adoc]: "more" is redundant (more than what?). "some depth" maybe?

File: docs/src/main/asciidoc/README.adoc [hunk 4]
```diff
@@ -4,10 +4,10 @@
 
 {repo_url}
 
-This article will explain in more depth how to tackle issues related to database compatibility with Spring Boot and
-database versioning tooling. We will present what can happen with your production applications if you try to perform
-such a deployment unprepared. We will also walk through the lifecycle of an application in order to perform zero
-downtime deployment and at the end of day apply a backwards incompatible database change in a backwards compatible way.
+This article will explain in depth how to tackle issues related to database compatibility and the deployment process.
+We will present what can happen with your production applications if you try to perform
+such a deployment unprepared. We will then wall through the steps in the lifecycle of an application that are necessary
```
Comment (dsyer) [inline docs/src/main/asciidoc/README.adoc]: "wall"

Comment (marcingrzejszczak) [inline docs/src/main/asciidoc/README.adoc]: fixed

File: README.adoc [hunk 5]
```diff
@@ -1,111 +1,240 @@
...
 
-How to write the code of your app to allow doing A/B testing or/and continuous deployment.
+:repo_url: https://github.com/marcingrzejszczak/database-update
 
-== Assumptions
+= Zero Downtime Deployment Database Issues
 
-In order for the process to be relatively simple the following assumptions need to take place
-while developing the app:
+{repo_url}
```
Comment (dsyer) [inline README.adoc:7]: Maybe move this to a second paragraph:

If you want to work though the code samples below, you will find everything you need in {repo_url}[GitHub].

Or something.

Comment (marcingrzejszczak) [inline README.adoc:7]: fixed

**Pull request merged**